data-jdbc/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     api projects.dataRuntime
 
     implementation mnSerde.micronaut.serde.support
+    implementation mnSerde.micronaut.serde.oracle.jdbc.json
```

Comment:
I don't think we want to add the oracle driver for everyone how is using `data-jdbc`